Libplanet.Unity/SwarmRunner.cs

```diff
@@ -37,6 +37,16 @@ public class SwarmRunner
             _cancellationTokenSource = new CancellationTokenSource();
         }
 
+        /// <summary>
+        /// EventHandler checked if IsPreloaded has changed.
+        /// </summary>
+        public delegate void EventHandler();
+
+        /// <summary>
+        /// Event checked if IsPreloaded has changed.
+        /// </summary>
+        public event EventHandler? IsPreloadedChanged;
```

Comment:
This EventHandler itself runs only once time when the preload is finished.
I don't think there will be a situation where I can receive an event that it has been preload changed.
Therefore, it seems better to delete the IsPreloaded that exists as the current variable and change it to EventHandler IsPreloaded.
Also, there seems to be little need to be public. better to change it to private.